src/main/java/net/schmizz/sshj/xfer/scp/SCPFileTransfer.java

```diff
@@ -28,18 +28,23 @@
         extends AbstractFileTransfer
         implements FileTransfer {
 
+    /** Default bandwidth limit for SCP transfert in Kbit/s (-1 means unlimited) */
```

Comment:
typo in comment (transfert). Also is it kilobit, or kilobyte? The difference is a factor 8.